_ml-commons-plugin/cluster-settings.md

```diff
@@ -492,3 +492,14 @@ plugins.ml_commons.rag_pipeline_feature_enabled: true
 
 - Default value: `true`
 - Valid values: `false`, `true`
+
```

Comment:
Please add a heading level 2 before this section that says what the setting does and a description of the setting.

Reply:
Thanks, add description.